packages/expo/src/singleton.ts

```diff
@@ -43,13 +41,6 @@ export function buildClerk({ key, tokenCache }: BuildClerkOptions): HeadlessBrow
 
       const jwt = await getToken(KEY);
       (requestInit.headers as Headers).set('authorization', jwt || '');
-
-      // Send some non-identifying headers that are useful for logging
-      (requestInit.headers as Headers).set('x-expo-execution-environment', Constants.executionEnvironment);
-      (requestInit.headers as Headers).set(
-        'x-expo-native-application-version',
-        Application.nativeApplicationVersion || 'NULL',
-      );
```

Comment:
@thiskevinwang If having this kind of information was useful, and don't wanna completely removed it, couldn't we just do this ?

```ts
if(!isValidBrowser()) {
      // Send some non-identifying headers that are useful for logging
      (requestInit.headers as Headers).set('x-expo-execution-environment', Constants.executionEnvironment);
      (requestInit.headers as Headers).set(
        'x-expo-native-application-version',
        Application.nativeApplicationVersion || 'NULL',
      );
}
```

of use `import { Platform } from 'react-native';` 

```ts
if (Platform.OS !== 'web') {
      // Send some non-identifying headers that are useful for logging
      (requestInit.headers as Headers).set('x-expo-execution-environment', Constants.executionEnvironment);
      (requestInit.headers as Headers).set(
        'x-expo-native-application-version',
        Application.nativeApplicationVersion || 'NULL',
      );
}
```

Reply:
@panteliselef Oh, smart thinking! But also...

> If having this kind of information was useful

...Turns out, it was not particularly useful so I'd say we just remove this!